css/style.css

```diff
@@ -727,23 +854,63 @@ details[open] summary::after {
   background-color: #28282C;
 }
 
+.form-message {
+  width: 100%;
+  max-width: 690px;
+  height: auto;
+  max-height: 536px;
```

Comment:
> 💛 1.20. Resizing <textarea> should not break the layout

This line helping me to break the layout by resizing the textarea. If you just remove this line, all will be good.

![image](https://github.com/user-attachments/assets/e11bd2d0-0dd5-414e-bf18-f41de05bae4c)